CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 ## Future
 - Fix false alarm on TRIO107 with checkpointing `try` and empty `finally`
+- Added `--no-checkpoint-warning-decorators`, 107/108 no longer ignores `asynccontextmanager` by default.
```

Comment:
I think the default value of this setting should ignore `asynccontextmanager`, so that if you don't explicitly configure anything you get the prior-to-this-PR behaviour.  This matches e.g. Coverage's ignore-patterns config; `# pragma: no cover` is ignored-by-default but if you're customizing ignore patterns you have to specify it manually.

Reply:
Ah sure, I was thinking ahead to future PR's handling `asynccontextmanager` and specifically turning off multiple-yield warnings and/or tweaking the functionality of checkpoint before `and`/`or` after yield. But future's the future